my-app/src/pages/Profile/Profile.tsx

```
@@ -595,13 +595,42 @@ const Profile = () => {
+          return /^(\+\d{1,2}\s?)?((\(\d{3}\))|\d{3})[\s.-]?\d{3}[\s.-]?\d{4}$/.test(phone);
+        };
+        const newErrors = { ...errors };
+        
+        if (name === "phone") {
+          if (value.trim() === "") {
+            newErrors.phone = "Phone is required";
+          } else if (countDigits(value) < 10) {
+            newErrors.phone = "Phone number must contain at least 10 digits";          } else if (!isValidPhoneFormat(value)) {
+            newErrors.phone = `"${value}" is an invalid format. Please see the i icon for allowed formats.`;
+          } else {
+            delete newErrors.phone;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)